### ISSUE TITLE
Que mensaje dara el bot a los nuevos integrantes?

### ISSUE THREAD

**Issue by german1608** [adeluccar/epale-bot#33]
Que cosas nos gustaria preguntarle al nuevo integrante? coloco aca algunas ideas

1) tabs vs spaces
2) editor
3) como llego a fcc
4) aspiraciones con fcc
5) experiencias anteriores

Llenen la lista. Ademas alguien debería de redactar tal mensaje. Podemos discutir eso por este issue

**Comment by criptodev** [comment]
Mis propuestas: 
*Personaje favorito de game of thrones, y su respectivo ¨winter is coming¨
*Toddy o Taco?
*A que te dedicas actualmente

Podemos hacer votaciones mas adelante y hacer un random de bienvenidas, para que cuando entre alguien nuevo le salgan una serie de preguntas al azar de una lista que haremos con las mejores preguntas.

**Comment by adeluccar** [comment]
El problema es que, ya Winter esta aquí. Jajaja!

Me gusta la idea de preguntas aleatorias. La otra cosa importante es balancear las preguntas para evitar sobrecargar a los recien llegados. Yo diría que máximo tres preguntas por run del bot.
Las primeras dos pueden ser fijas, y una tercera de j*da aleatoria.

¿Que creen?

**Comment by adeluccar** [comment]
Propongo:

```
¡Bienvenid@ a la comunidad de fCC Caracas!
Cuéntanos sobre ti :)
¿Estas haciendo los cursos de fCC?
¿Cuales son tus expectativas?
¿Tabs o Spaces? XD # Esta ultima es aleatoria
```

**Comment by criptodev** [comment]
Me agrada tres preguntas, aunque me parece que las preguntas fijas deben ir mas orientadas a conocer a la persona que a su curriculum en fcc. Pero si, es buena idea.

**Comment by german1608** [comment]
que tal 3 preguntas fijas que se orienten a conocer mas al nuevo integrante y 2 pregutas random (tipo tabs, beisbol, etc)

**Comment by adeluccar** [comment]
@german1608 También es válido. Me parece bien. Dejaré este issue abierto hasta que lleguemos al punto donde realmente tengamos que tomar esta decisión.

**Comment by IsaacYAGI** [comment]
También me gustó la idea de las tres preguntas fijas. Las que ya están creo que están bastante bien:

1. Como conociste el grupo.
2. Cuales son tus expectativas.
3. Cuales son tus intereses.
4. RANDOM
5. RANDOM.

**Comment by german1608** [comment]
bueno entonces cuando empezemos a echar codigo podemos hacer PR solo para agregar (o corregir) preguntas RANDOM. Las preguntas fijas si las podemos decidir aqui

**Comment by Alvillarroel** [comment]
No se puede hacer para que en vez de que responda esas preguntas cerradas, tuviera un form para que escojiera la respuesta. 
algo asi como Tabs vs Spaces? (opcion tab) (opcion Spaces)

**Comment by german1608** [comment]
es buena idea. Podriamos hacer que el bot llame a VoteBot para decidir eso o algo asi idk.

**Comment by Anibalismo** [comment]
Entendiendo que somos un grupo informal y aclarando que me gusta juego de trono tanto como a cualquiera acá, el mensaje de bienvenida estático debería de representar al grupo, y aclarar la primera duda de los nuevos ingresos (que es "en qué peo me acabo de meter"). Algo en las líneas de (tomado de adeluccar) 

> ¡Bienvenid@ a la comunidad de fCC Caracas! Somos un grupo para compartir y contrastar experiencias en el aprendizaje de tecnologías web

**Comment by adeluccar** [comment]
Epale @Alvillaruel gracias por la idea. Bajo los parámetros que estamos considerando para el bot (hook), ponerlo a votar no parece posible. El bot de bienvenida solo puede retornar mensajes.

De todas formas, vamos a tomar en cuenta la sugerencia, y ver si es factible o no.

**Comment by adeluccar** [comment]
@Anibalismo Estoy totalmente de acuerdo con lo que expones. Si bien hay lugar para la joda, creo que la primera experiencia debería ser más en serio, y menos en joda. Por eso hacía hincapié en el número de preguntas. Nadie entra a una habitación a recibir un interrogatorio.

**Comment by adeluccar** [comment]
Por ejemplo, la pregunta de cómo conociste el grupo es prácticamente innecesaria. No son muchos los sitios donde se está haciendo publicidad del grupo y no sirve mucho propósito.

**Comment by german1608** [comment]
Que les parece si creamos un archivo que contenga el mensaje? algo tipo `mensaje-bienvenida.md`.
Asi si tenemos algo base y pues lo vamos modificando en el transcurso de tiempo.

**Comment by adeluccar** [comment]
@german1608 No es necesario. Literalmente va a ser una variable en el programa final. Recuerda el principio DRY (Don't Repeat Yourself). Esa variable la pueden cambiar con PRs igualito que ese documento, pero mantener ambos actualizados agrega una capa de abstracción con la que puedes equivocarte (pues tienes que recordar cambiar el mismo mensaje en dos sitios distintos).

**Comment by bettoarcaya** [comment]
El bot podria dar una bienvenida formal al grupo, invitar a realizar preguntas sobre dudas sobre FCC, etc. propongo como pregunta el porque escogió FCC para aprender tecnologias web, las otras preguntas pueden ser aleatorias.

**Comment by criptodev** [comment]
@Anibalismo podemos usar el voteBot para determinar los random questions a usar segun decida la comunidad, pero no es necesario llamar al voteBot cada vez que entre alguien nuevo, puesto que de por si este problema de los random questions se resuelve super facil con una funcion random aplicada a la lista de preguntas ( sean obj o array).

**Comment by criptodev** [comment]
Yo digo que no mas de tres preguntas, cinco preguntas ya es interrogatorio, la cosa es determinar cuantas de las tres preguntas seran random y cuantas fijas.

**Comment by adeluccar** [comment]
@criptodev Estoy de acuerdo contigo. Tres preguntas máximo con una random (pues es fácil sacar una sola de un array de preguntas sin tener que revisar si es duplicada o algo). cc @Anibalismo @bettoarcaya

**Comment by adeluccar** [comment]
El bot ya esta operativo en la sala. Voy a postear las instrucciones en https://github.com/adeluccar/guelcome-bot para que otros grupos puedan hacer lo mismo, pero podemos continuar conversando el mensaje del nuestro aquí.

Por los momentos el bot solo dice: `Welcome fulano`

cc @german1608 @Anibalismo @IsaacYAGI @criptodev @cristche @bettoarcaya
